build.gradle

```diff
@@ -0,0 +1,158 @@
+import org.apache.tools.ant.filters.ReplaceTokens
+
+
+plugins {
+    id 'war'
+    id 'java'
+    // IntelliJ IDEA plugin here to allow integration tests to appear properly in IDEs.
+    id 'idea'
+}
+
+repositories {
+    mavenLocal()
+    jcenter()
+
+    maven {
+        url = 'http://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/m2repo'
+    }
+}
+
+dependencies {
+
+    runtime 'jstl:jstl:1.+'
+
+    implementation 'org.apache.logging.log4j:log4j-core:[2.13.0,3.0)'
+    implementation 'org.apache.logging.log4j:log4j-slf4j-impl:[2.13.0,3.0)'
+    implementation 'com.opencsv:opencsv:[5.1,6.0)'
+    implementation 'org.opencadc:cadc-web-util:[1.2.7,1.3.0)'
+    implementation 'commons-net:commons-net:3.6'
+
+    runtime 'javax.servlet.jsp.jstl:jstl-api:1.2'
+    runtime 'javax.servlet:jstl:1.2'
+    runtime 'org.apache.taglibs:taglibs-standard-impl:1.2.5'
+
+    testCompile 'ca.nrc.cadc:web-test:[2.0, )'
+    testCompile 'org.seleniumhq.selenium:selenium-java:3.+'
+}
+
+sourceCompatibility = '1.8'
+group = 'ca.nrc.cadc'
+version = '1000'
+
+ext {
+    intTest_user_name = 'CADCtest'
+    intTest_user_password = 'sywymUL4'
+
+    docker_image_name = 'mach309.cadc.dao.nrc.ca:5000/science-portal'
```

Comment:
This image will not run in CI/CD.  Is that the intent?

Reply:
No, not the intent. Typo!

Reply:
I removed the docker references for now, as they can be added in later easily and in the spirit of this being a first fly that isn't intended to be run in a docker container, making it work is getting in the way of having it be deployed. If needed, it can be implemented in the next story for this app.